javascript/javascript2/week2/main/week1.js

```diff
@@ -0,0 +1,110 @@
+//This file is NOT part of week2 project
+//All updates are in week2.js
+
+const div = document.createElement("div");
+div.classList = "card";
+document.body.appendChild(div);
+div.setAttribute("id", cards[1].id);
+
+div.addEventListener("click", function () {
+  if (div.className === "card") {
+    div.setAttribute("style", 'background-image: url("' + cards[1].url + '");');
+    div.classList = "card_flipped card_is-flipped card_changed-color";
```

Comment:
This is not valid code. Make sure to have a look at the documentation of `classList`: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList